basebinaries/libjailbreak/libjailbreak.h

```diff
@@ -0,0 +1,18 @@
+//
+//  libjailbreak.h
+//  libjailbreak
+//
+//  Created by Jamie Bishop on 29/01/2018.
+//  Copyright © 2018 Electra Team. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+struct __attribute__((__packed__)) JAILBREAKD_ENTITLE_PID {
+    uint8_t Command;
+    int32_t Pid;
+};
+
+void entitle(void);
+void platformize(void);
+void fix_setuid(void);
```

Comment:
we should give these public APIs a name prefix (i.e. `jb_entitle`) in case someone uses these names in their program